### ISSUE TITLE
cacert.py

### ISSUE THREAD

**Issue by dsapikas** [pusher/pusher-http-python#59]
As i informed you in person, when i installed the library (not sure if it was from pip or easy_install), i got an error for missing the file "cacert.py", when i copied it from this repository, worked just fine.

**Comment by vetal4444** [comment]
Did you mean?

    lib/python3.4/site-packages/pusher/cacert.pem

If yes, I confirm this. Install package with 

    pip install pusher

**Comment by tgwizard** [comment]
I just installed pusher via pip and now I get this error:

```
-- snip --
File "/path/to/virtualenv/local/lib/python2.7/site-packages/pusher/http.py", line 22, in __call__
  return self.pusher.http.send_request(self.make_request(*args, **kwargs))
File "/path/to/virtualenv/local/lib/python2.7/site-packages/pusher/requests.py", line 37, in send_request
  **self.options
File "/path/to/virtualenv/local/lib/python2.7/site-packages/requests/sessions.py", line 465, in request
  resp = self.send(prep, **send_kwargs)
File "/path/to/virtualenv/local/lib/python2.7/site-packages/requests/sessions.py", line 573, in send
  r = adapter.send(request, **kwargs)
File "/path/to/virtualenv/local/lib/python2.7/site-packages/requests/adapters.py", line 431, in send
  raise SSLError(e, request=request)
SSLError: [Errno bad ca_certs: u'/path/to/virtualenv/local/lib/python2.7/site-packages/pusher/cacert.pem'] [('system library', 'fopen', 'No such file or directory'), ('BIO routines', 'BIO_new_file', 'no such file'), ('x509 certificate routines', 'X509_load_cert_crl_file', 'system lib')]
```

**Comment by tgwizard** [comment]
And setting `ssl=False` when creating the `Pusher` client makes it work, but that I don't want to do :disappointed:.

**Comment by jpatel531** [comment]
I just published 1.2.1, which resolves this. Thanks again @dsapikas  for bringing this up!
